content/applications/services/project/tasks/task_stages_statuses.rst

```diff
@@ -0,0 +1,71 @@
+========================
+Task stages and statuses
+========================
+
+Task stages
+===========
+
+Task stages are displayed as columns in the project's Kanban view, and allow you to update the
+progress of its tasks with a drag-and-drop. In most projects, the stages will be akin to **New**,
+**In progress**, **Backlog**, etc.
+
+By default, task stages are project-specific but can be shared across multiple projects that follow
+the same workflow.
+
+Creating task stages
+--------------------
+
+Odoo Project doesn’t provide default stages but instead allows you to create custom stages tailored
+to your specific business needs. You are prompted to do so immediately after creating a new project.
```

Comment:
I think it's be nice to add a ref to the Configuration section in the Project management doc